awsenv/replacer.go

```diff
@@ -35,22 +36,24 @@ type LimitedParamsGetter interface {
 // given value prefix, using the given ParamsGetter.
 //
 // NewReplacer will panic if envValuePrefix is the empty string.
-func NewReplacer(envValuePrefix string, ssm ParamsGetter) *Replacer {
+func NewReplacer(envValuePrefix string, unsetNotFound bool, ssm ParamsGetter) *Replacer {
```

Comment:
This changes the function signature, and thus would represent a breaking change. We could instead just expose `UnsetNotFound` on the Replacer struct, so that it can be set to true _after_ calling `NewReplacer`; adding new fields to a struct is not considered a breaking change by the Go community.